lib/react_phoenix/react_io/path_finder.ex

```diff
@@ -5,10 +5,16 @@ defmodule ReactPhoenix.ReactIo.PathFinder do
   @phoenix_1_2_react_stdio_path Path.join(~w(#{@app_dir} .. .. node_modules .bin react-stdio))
   @local_react_stdio_path Path.join(~w(node_modules .bin react-stdio))
 
+  require Logger
+
   def react_stdio_path(), do: react_stdio_path(default_locations())
   def react_stdio_path(locations) when is_list(locations), do: do_react_stdio_path(locations)
 
-  defp do_react_stdio_path([]), do: raise("#{ReactPhoenix}: An installation of react-stdio cannot be found. Please run `npm install` and/or set the :react_stdio_path config value for :react_pheonix in your application.")
+  defp do_react_stdio_path([]) do
+    msg = "#{ReactPhoenix}: An installation of react-stdio cannot be found. Please run `npm install` and/or set the :react_stdio_path config value for :react_pheonix in your application.  Then RECOMPILE this package (mix deps.clean react_phoenix && mix deps.compile).  Server-side rendering won't work until you fix this issue.  If you only use client-side rendering, you can ignore this."
+    Logger.warn(msg)
+    nil
```

Comment:
Let's keep this error message with some modifications:

1. Please only use 1 space after a period.
2. Surround the recompile calls with backticks like "npm install" has
3. Split the message over several lines to keep line length down. Perhaps in this way:
   ```
   "ReactPhoenix: An installation..."
   <> " more info"
   <> " more info"
   |> Logger.warn()
   nil
   ```